### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by kacperpONS**
# Motivation and Context

Added the gtag.js script and the gtag_id to enable GA

# What has changed

Added the script and the gtag_id
Cleaned up the templates

# How to test?

Make sure it looks ok, will be tested once deployed to preprod.

# Links

https://jira.ons.gov.uk/browse/SDCSRM-626

# Screenshots (if appropriate):

### CODE REVIEW

File: rh_ui/templates/base.html [hunk 1]
```diff
@@ -4,8 +4,28 @@
...
 
-{% set cookiesPageURL = url_for('i18n.info_pages_bp.cookies') %}
+{% set g_tag_id = config.GOOGLE_TAG_ID %}
+
+
+{% block head %}
+    {% if g_tag_id %}
+        {% set script_src_url = "https://www.googletagmanager.com/gtag/js?id=" + g_tag_id %}
+        <!-- Google tag (gtag.js) -->
+        <script async src=sript_src_url></script>
```
Comment (ryangrundy7) [inline rh_ui/templates/base.html:14]:
```suggestion
        <script async src=script_src_url></script>
```

File: rh_ui/templates/base.html [hunk 2]
```diff
@@ -4,8 +4,28 @@
 {%- block preHeader -%}
 {%- autoescape false -%}
 
-{% set cookiesPageURL = url_for('i18n.info_pages_bp.cookies') %}
+{% set g_tag_id = config.GOOGLE_TAG_ID %}
+
+
+{% block head %}
+    {% if g_tag_id %}
+        {% set script_src_url = "https://www.googletagmanager.com/gtag/js?id=" + g_tag_id %}
```
Comment (ryangrundy7) [inline rh_ui/templates/base.html:12]: Should this be disabled if the user doesn't want additional cookies? Can't see anything blocking it at the moment

File: rh_ui/templates/base.html [hunk 3]
```diff
@@ -4,8 +4,28 @@
 {%- block preHeader -%}
 {%- autoescape false -%}
 
-{% set cookiesPageURL = url_for('i18n.info_pages_bp.cookies') %}
+{% set g_tag_id = config.GOOGLE_TAG_ID %}
+
+
+{% block head %}
```
Comment (ryangrundy7) [inline rh_ui/templates/base.html:10]: You've got a head block inside the preHeader which is causing duplication

File: rh_ui/templates/base.html [hunk 4]
```diff
@@ -1,25 +1,50 @@
...
-{%- block preHeader -%}
-{%- autoescape false -%}
+{% block head %}
+    {% set g_tag_id = config.GOOGLE_TAG_ID %}
+    {% if g_tag_id %}
+        {% set script_src_url = "https://www.googletagmanager.com/gtag/js?id=" + g_tag_id %}
+
+        <!-- Google tag (gtag.js) -->
+        <script nonce="{{ csp_nonce() }}" async src={{ script_src_url }}></script>
+        <script nonce="{{ csp_nonce() }}"w>
```
Comment (ryangrundy7) [inline rh_ui/templates/base.html:11]:
```suggestion
        <script nonce="{{ csp_nonce() }}">
```

**Review by ryangrundy7 — APPROVED** [review]

**Review by hbrace — APPROVED** [review]

**Pull request merged**